src/dtx/dtx_internal.h

```diff
@@ -143,8 +143,8 @@ extern uint32_t dtx_agg_thd_age_lo;
 extern uint32_t dtx_batched_ult_max;
 
 /* The threshold for using helper ULT when handle DTX RPC. */
-#define DTX_RPC_HELPER_THD_MIN	18
-#define DTX_RPC_HELPER_THD_DEF	(DTX_THRESHOLD_COUNT + 1)
```

Comment:
it means that we intend to offload DTX RPC to helper xstream correct?

Reply:
Yes, we intent to sent DTX RPC via helper XS.